src/DefaultEditMenu.js

```diff
@@ -14,67 +14,20 @@ const DefaultEditMenu = React.createClass({
         electron: React.PropTypes.object.isRequired
     },
 
-    getWindow() {
-        const { electron } = this.context;
-        return electron.remote.getCurrentWindow();
-    },
-
-    onUndo() {
-        this.getWindow().webContents.undo();
-    },
-
-    onRedo() {
-        this.getWindow().webContents.redo();
-    },
-
-    onCut() {
-        this.getWindow().webContents.cut();
-    },
-
-    onCopy() {
-        this.getWindow().webContents.copy();
-    },
-
-    onSelectAll() {
-        this.getWindow().webContents.selectAll();
-    },
-
     render() {
         const { children } = this.props;
 
         return (
             <MenuItem id="edit" label="Edit">
-                <MenuItem
-                    label="Undo"
-                    accelerator="CmdOrCtrl+Z"
-                    onClick={this.onUndo}
-                    />
-                <MenuItem
-                    label="Redo"
-                    accelerator="Shift+CmdOrCtrl+Z"
-                    onClick={this.onRedo}
-                    />
+                <MenuItem role="undo" />
+                <MenuItem role="redo" />
+                <MenuItem.Separator />
+                <MenuItem role="cut" />
+                <MenuItem role="copy" />
+                <MenuItem role="paste" />
+                <MenuItem role="delete" />
                 <MenuItem.Separator />
```

Comment:
Why is there a separator between `delete` and `selectall` ?

The menu for chrome is:

<img width="263" alt="screen shot 2017-02-18 at 16 12 27" src="https://cloud.githubusercontent.com/assets/845425/23094220/56c58864-f5f5-11e6-91d2-06660757eeed.png">